precode.sh

```diff
@@ -7,37 +7,73 @@
 #   dir3
 #       dir4
 
+mkdir task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Используя флаг `-p` можно не указывать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```